debug.log

```diff
@@ -0,0 +1 @@
+[0122/104722.476:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
pl. remember to delete this if not required (it is a debug log, so can be removed)